styles.css

```diff
@@ -0,0 +1,59 @@
+* {
+  background-color: #FFF;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-list {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  list-style: none;
+  padding: 1rem;
+  margin: 1rem;
+}
+
+#nav-bar-btn {
+  border: none;
+}
+
+#nav-bar-p {
+  font-family: 'Inter';
+  font-weight: 600;
+  font-size: 1.7rem;
+  color: #28352F;
+}
+
+.heading-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  text-align: center;
+  font-family: 'Inter';
+  font-style: normal;
+  height: 750px;
+  background-image: url('images/header-illsutration-mobile@2x.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
```

Comment:
- [ ] kindly make sure you use the right margins, the margins around your heading are not matching the Figma design, ability to replicate the design is one of the core requirements for this task.

- [ ] Please make sure the font sizes and font styles match the Figma designs.

- [ ] The background is not visible in the mobile version, Kindly make sure that the background image are showing in the mobile view

your project:
![image](https://user-images.githubusercontent.com/43564324/164347846-baca9eee-08c3-483e-b839-91d6bd74cd8a.png)

figma:
![image](https://user-images.githubusercontent.com/43564324/164348134-7a69f632-aa33-48cc-b899-a1f13f66b637.png)